sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/models/ChatEventId.java

```diff
@@ -16,6 +16,5 @@
     chatThreadDeleted,
     chatThreadPropertiesUpdated,
     participantsAdded,
-    participantsRemoved,
-    connectionChanged;
+    participantsRemoved
```

Comment:
the naming convention for enum value is - all caps with appropriate use of underscore.. e.g. `CHAT_MESSAGE_RECEIVED`

Reply:
Is not it better to call this `ChatEventKind`? It is not really the id of an event. Id implies identification and uniqueness.